File: BackEnd/ExamManagement/DPD.HR.Business.Dto/Dto/TestScheduleDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DPD.HumanResources.Dtos.Dto
{
    public class TestScheduleDto
    {
        public Guid Id { get; set; }
        public string? TestScheduleName { get; set; }
        public bool IsActive { get; set; }
        public DateTime? CreatedDate { get; set; }
        public Guid? IdExam { get; set; }
        public Guid? IdExamSubject { get; set; }
        public DateTime? FromDate { get; set; }
        public DateTime? ToDate { get; set; }
        public bool IsDeleted { get; set; }
        public bool OrganizeFinalExams { get; set; }
        public string? Note { get; set; }
    }
}

File: FrontEnd/src/models/TestScheduleDTO.ts
export interface TestScheduleDTO {
  examTypeName: string;
  id: string;
  testScheduleName?: string | null;
  isActive: boolean;
  createdDate?: Date | null;
  idExam?: string | null;
  idExamSubject?: string | null;
  fromDate?: Date | null;
  toDate?: Date | null;
  isDeleted: boolean;
  organizeFinalExams: boolean;
  note?: string | null;
}
